pimd/pim6_cmd.c

```diff
@@ -46,6 +46,13 @@
 #include "pimd/pim6_cmd_clippy.c"
 #endif
 
+static struct cmd_node debug_node = {
+	.name = "debug",
+	.node = DEBUG_NODE,
+	.prompt = "",
```

Comment:
I don't think we set `prompt` in other places where the node is not actually used / reachable.

Reply:
This has been used like this throughout FRR code for debug commands.